resources/profiles/Kingroon/filament/PRINTALOT - PLA - ARTDUE - VERDE-AMARILLO @kp3s pro v2 @0.4.json

```diff
@@ -0,0 +1,28 @@
+{
+  "additional_cooling_fan_speed": ["100"],
+  "default_filament_colour": ["#F0FC0C"],
+  "enable_overhang_bridge_fan": ["0"],
+  "enable_pressure_advance": ["0"],
+  "filament_flow_ratio": ["0.9"],
+  "filament_max_volumetric_speed": ["37"],
+  "filament_retraction_length": ["0.8"],
+  "filament_retraction_speed": ["45"],
+  "filament_settings_id": [
+    "PRINTALOT - PLA - ARTDUE - VERDE-AMARILLO @kp3s pro v2 @0.4"
+  ],
+  "filament_z_hop": ["nil"],
+  "from": "User",
+  "inherits": "Kingroon Generic PLA",
+  "name": "PRINTALOT - PLA - ARTDUE - VERDE-AMARILLO @kp3s pro v2 @0.4",
```

Comment:
Can we make name a bit more simpler?
Something like `Printalot PLA - ART DUE` ?
If this is filament profile is very specifically designed for one printe model, we can name it like this:
`Printalot PLA ARTDUE @KP3S PRO V2 0.4 nozzle` or `Printalot PLA ARTDUE @KP3S PRO V2 (0.4 nozzle)`
But usually I suggest we keep them genertic instead of for one specific printer model.
In this case, it's compatible with two printers, so we can the genertic name. `Printalot PLA ARTDUE'